keps/sig-scheduling/20180409-scheduling-framework.md

```diff
@@ -264,6 +268,31 @@ state, it will either trigger [Un-reserve](#un-reserve) plugins (on failure) or
 
 *Note: This concept used to be referred to as "assume".*
 
+### Pre-Preempt
+
+This is an informational extension point. Plugins will be called with a pod which was unscheduled and eligible to preemption. A plugin may use this data to update internal state or to generate logs/metrics.
+
+### Preempt
+
+The plugin provides an extension point for selecting the victim pods for preemption.  Only one preempt plugin may be enabled at a time.  
+
+The selection process are performed in cooperating with scheduler and preempt plugin as follows.
+
+1. Scheduler firstly filters _potential victim nodes_ that satisfies the following two conditions.  It contains _potential victim pods_ that are allowed to be preempted by the preemptor pod according to their `spec.preemptionPolicy`.  The other is the preemptor pod can be schedulable when all the potential victim pods are assumed to be preempted.
```

Comment:
according to its preemptionPolicy.

The preemption policy affects the preemptor, not the victims.

Reply:
Yeah, I understood this.  I will re-phrase this sentence so that preempotor's `spec.preemptionPolicy` affects this phase.  I wanted to say no victims will be selected if preemptor's `spec.preemptionPolicy==Never`.